src/cpp/transport/UDPTransportInterface.cpp

```diff
@@ -451,8 +451,27 @@ bool UDPTransportInterface::ReleaseInputChannel(const Locator_t& locator, const
 
             // We then send to the address of the input locator
             auto destinationEndpoint = GenerateLocalEndpoint(locator, port);
-            socket.send_to(asio::buffer("EPRORTPSCLOSE", 13), destinationEndpoint);
 
+#ifdef WIN32
```

Comment:
I think it is better to always have this behavior, as I've also seen similar error output [on Mac](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/1151/consoleFull). 

If no error is returned, then there is no problem. If it is an expected error, then there is no problem. So I think we can have this for all platforms.

Reply:
done!